app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "cp":
+        raise ValueError("Invalid command."
+                         "Use format: cp source_file target_file")
```

Comment:
There is a missing closing parenthesis for the `ValueError` exception. Ensure that the parentheses are correctly closed to avoid syntax errors.